cluster-autoscaler/main_test.go

```diff
@@ -17,24 +17,25 @@ limitations under the License.
 package main
 
 import (
-	"github.com/stretchr/testify/assert"
-	"k8s.io/autoscaler/cluster-autoscaler/context"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/autoscaler/cluster-autoscaler/config/static"
```

Comment:
If you're already re-organizing imports with the convention we use it should be:
```
"testing" 

"k8s.io/autoscaler/cluster-autoscaler/config/static"

"github.com/stretchr/testify/assert"
```
Admittedly it looks kinda strange with just one import in each group :)

Reply:
Done